src/components/overview.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
Out of scope, remove it. You will work on it when you start working on TV Shows task.

Reply:
Components that behave like routes should be inside views folder not components folder.